packages/mgt-components/src/components/mgt-person/mgt-person.scss

```diff
@@ -64,6 +63,7 @@ mgt-person .person-root {
   position: relative;
   display: flex;
   align-items: center;
+  color: var(--person-color, $color);
```

Comment:
Looks like the person component gets its main color from our common variables, looks like `--color-sub1` changes the main color. 

I don't think we need to introduce yet another var here. My proposal is to:
* no need to make any changes in this file (mgt-person.sass) as it looks like the main color is using the `$color` variable coming from the shared-sass-variables file
* update the $color variable to use `--color` instead of `--color-sub1` as that makes more sense for a global var to control color across the toolkit. 
* use the `--color` var in the `mgt-login.ts` file to set the color of the `mgt-person`

We should also document all of the vars in the `shared-sass-variables.scss` page as those are variables used across components but are not documented.

Thoughts?